notifications/rest/bindings/create-binding/create-binding.5.x.rb

```diff
@@ -6,7 +6,7 @@
 auth_token = 'your_auth_token'
 client = Twilio::REST::Client.new account_sid, auth_token
 
-service = client.notify.services('ISxxx')
+service = client.notify.v1.services('ISxxx')
```

Comment:
@hortega according to the curl sample please use `ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX`